simapp/app.go

```diff
@@ -142,6 +144,7 @@ var (
 		vesting.AppModuleBasic{},
 		tokenmodule.AppModuleBasic{},
 		collectionmodule.AppModuleBasic{},
+		//ordamodule.AppModuleBasic{},
```

Comment:
Why is this and the following commented out?

https://github.com/Finschia/finschia-sdk/pull/1032/files#diff-8d1ca8086ee74e8f0490825ba21e7435be4753922192ff691311483aa3e71a0aR398

Reply:
because rollup module isn't implemented yet. So orda keeper inject that keeper as `nil`. So some tests are panics because of that reason.